ruby_on_rails/app_initialisation.md

```diff
@@ -106,4 +106,22 @@ config.log_level = :warn # change
 config.action_view.raise_on_missing_translations = true # uncomment
 ```
 
+* Enable the default Content Security Policies in `config/initializers/content_security_policy.rb`:
+
+```ruby
+Rails.application.config.content_security_policy do |policy|
+  policy.default_src :self, :https
+  policy.font_src    :self, :https, :data
+  policy.img_src     :self, :https, :data
+  policy.object_src  :none
+  policy.script_src  :self, :https
+  policy.style_src   :self, :https
+
+  policy.report_uri ENV['CSP_REPORT_URI']
+end
```

Comment:
is this configuration correct in the light of what happened to bankentracker?

Reply:
yes, it is. The problem is the nonce generator